cmd/license-detector/main_test.go

```diff
@@ -6,21 +6,20 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"gopkg.in/src-d/go-license-detector.v2/licensedb"
 )
 
-func TestMain(t *testing.T) {
+func TestCmdMain(t *testing.T) {
```

Comment:
Changed the method name to avoid clashing with https://golang.org/pkg/testing/#hdr-Main